db/migrate/20160921115843_create_offsprings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOffsprings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :offsprings do |t|
```

Comment:
No, you don't create different tables, you create a table with everything you need